rasa/core/domain.py

```diff
@@ -47,6 +52,11 @@ def __str__(self):
         return bcolors.FAIL + self.message + bcolors.ENDC
 
 
+class SessionConfig(NamedTuple):
+    session_length: Union[int, float]
```

Comment:
```suggestion
    session_length: float
```
`float` implies `int`, see https://www.python.org/dev/peps/pep-0484/#the-numeric-tower

Reply:
thanks, didn't know that! I thought I was clever because I have seen that in Rasa X 🙈